packages/containers/psammead-timestamp-container/src/relativeTimestamp.js

```diff
@@ -0,0 +1,36 @@
+const constructTimestamp = timeDiffMins => {
+  let timeDiff;
+  let unit;
+
+  if (timeDiffMins >= 1440) {
+    timeDiff = parseInt(timeDiffMins / 1440, 10);
+    unit = timeDiff > 1 ? 'days' : 'day';
+  } else if (timeDiffMins >= 60) {
+    timeDiff = parseInt(timeDiffMins / 60, 10);
+    unit = timeDiff > 1 ? 'hours' : 'hour';
```

Comment:
since the content of these `if/else` statements has a bit of duplication we could create a function like this:
```js
const pluralise = (timeDiffMins, comparator, singular) => {
    timeDiff = parseInt(timeDiffMins / comparator, 10);
    unit = timeDiff > 1 ? `${singular}s` : singular;
}
```

then you could call it
`pluralise(timeDiffMins, oneHour, 'hour')`
in each of the above scenarios. I think this would reduce the cognitive complexity.